src/apps/Agreement/DisputableApps/DisputableApps.js

```diff
@@ -0,0 +1,221 @@
+import React from 'react'
+import {
+  DataView,
+  ContextMenu,
+  ContextMenuItem,
+  useTheme,
+  GU,
+  useLayout,
+  Help,
+  TokenBadge,
+} from '@aragon/ui'
+import { KnownAppBadge } from '../../../templates/kit'
+import InfoField from './../InfoField'
+import { AppItemsType } from './../prop-types'
+import SubtleLabel from './SubtleLabel'
+import TokenAmount from './TokenAmount'
+
+const EXPANDABLE_ROW_GAP = `${3 * GU}px`
+
+const DisputableApps = React.memo(({ items }) => {
+  const { layoutName } = useLayout()
+  const theme = useTheme()
+
+  return (
+    <DataView
+      fields={[
+        { label: 'App', priority: 1, childStart: true },
+        { label: 'Actions', align: 'left' },
+      ]}
+      entries={items}
+      renderEntry={entry => renderEntry(entry, layoutName)}
+      renderEntryExpansion={entry => renderEntryExpansion(entry, layoutName)}
+      renderEntryActions={entry => renderEntryActions(entry, theme)}
+    />
+  )
+})
+
+function renderEntry({ allowedActions }, layoutName) {
+  return [
+    <div
+      css={`
+        display flex;
+        align-items: center;
+
+        /* Height must match expansion button to align nicely */
+        ${(layoutName === 'medium' || layoutName === 'large') &&
+          'height: 32px;'}
```

Comment:
We should probably use a GU for this value instead, but also curious if there was something we could do to avoid forcing this here

Reply:
Improving `DataView` to make the expansion content sit optically aligned to the icon would do it. In lieu of that this is the next best option I think.

Reply:
Gotcha—should we create an issue to solve this in aragonUI at a later time?